src/main/java/com/thoughtworks/gauge/Table.java

```diff
@@ -95,8 +102,90 @@ public void addRow(List<String> row) {
     
     @Override
     public String toString(){
-        return "Table{" +
-                        tableRows.toString() +
-                        '}';
+        int maxStringLength=getMaxStringLength();
```

Comment:
Hi @apoorvam 

- I know that's a lot of codes only to format properly, could be more concise with java 8 lambda 😄 
- I've took the overall max could have been max per column but I guess it's good enough for now
- I've not done TableRow toString changes